nix_update/__init__.py

```diff
@@ -101,6 +108,11 @@ def git_commit(git_dir: str, attribute: str, package: Package) -> None:
             )
 
 
+def write_commit_message(path: str, attribute: str, package: Package) -> None:
+    with open(path, "w") as f:
```

Comment:
It would be more fitting if we write this into the cache directory where we already write the report and the two nix files.

Reply:
I think it's more versatile this way. An alternative wood be if nix-update had a formatted stdout, but this would require some work.